packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -95,28 +100,29 @@ const Bookmarks = () : JSX.Element => {
       logger.error('failed to fetch data', error);
       toastError(error, 'Error occurred in bookmark page list');
     }
-  }, [currentUser]);
+  }, [currentUser, page]);
 
   useEffect(() => {
     getMyBookmarkList();
   }, [getMyBookmarkList]);
 
   const renderBookmarksItem = () => {
-    if (pages.length === 0) {
+    if (pages?.length === 0) {
       return (
         <h3 className="pl-3">
           { t('No bookmarks yet') }
         </h3>
       );
     }
-    return <BookmarksItem pages={pages} />;
+    return <BookmarksItem pages={pages} refreshBookmarkList={getMyBookmarkList} />;
```

Comment:
`getMyBookmarkList` should be replaced `mutate` function of swr.

Also, the props name `refreshBookmarkList` is not appropriate. it should be like `onBar`.